src/foam/nanos/jetty/HttpServer.js

```diff
@@ -99,10 +107,15 @@ foam.CLASS({
       class: 'FObjectProperty',
       of: 'foam.nanos.logger.Logger',
       visibility: 'HIDDEN',
-      javaFactory: `
+      javaGetter: `
```

Comment:
ok, so getX().get("logger") is not null when we call javaFactory but:
in DAOLogger getDelegate is null though it shouldn't be and it isn't in other places. It is localLogMessageDAO.

so logger == null doesn't fix it. javaGetter fixes it but it doesn't look good

Reply:
Ok, just explicitly use a StdoutLogger from here, and we'll look into setup order later.

Reply:
the problem occur after http server record update